feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/thread/MetisThreadUi.kt

```diff
@@ -267,7 +278,13 @@ private fun PostAndRepliesList(
         )
     }
 
-    ProvideMarkwon {
+    val context = LocalContext.current
+    var imageLoader: ImageLoader? by remember { mutableStateOf(null) }
+    LaunchedEffect(rememberCoroutineScope()) {
+        imageLoader = imageLoaderCreation(context).await()
+    }
+
+    ProvideMarkwon(imageLoader = imageLoader) {
```

Comment:
That is a valid question. Unfortunately, I don't know. I tested it without having it twice, and there is no difference so that we could eliminate it in the thread UI. @TimOrtel, what is your opinion on this?

Reply:
I think it should be fine.